lms/djangoapps/course_api/tests/test_views.py

```diff
@@ -201,6 +201,36 @@ def test_filter(self):
             response = self.verify_response(params=params)
             assert {course['course_id'] for course in response.data['results']} == {str(course.id) for course in expected_courses}, f'testing course_api.views.CourseListView with filter_={filter_}'  # pylint: disable=line-too-long
 
+    def test_get_when_no_permission_then_filters_correctly(self):
+        """
+        Given a user that is instructor in a course
+        And another course he is not an instructor
+        When get
+        Then only the first course is returned
+        """
+
+        # Create a second course to be filtered out of queries.
+        self.create_course(course='i-dont-want-to-see-you-course')
```

Comment:
Minor: Same comment as above.

```suggestion
        self.create_course(course='not-included-course')
```